### ISSUE TITLE
🎨 Top links are always displayed in white, no color change on hover

### ISSUE THREAD

**Issue by fuse314** [nunocoracao/blowfish#813]
**Describe the bug**
The top links are always displayed in the color white (dark theme) or black (light theme).
The "p" attribute within the link does have the following classes ``text-base font-medium text-gray-500 hover:text-gray-900``, but the according css declarations for ``text-gray-500`` and ``hover:text-gray-900`` are missing in the generated CSS file ``main.bundle.min*.css``.

**To Reproduce**
1. Go to https://blowfish.page/
2. Open web developer console in browser (usually F11 or Ctrl+Shift+I)
3. Inspect the links to "/docs/" and "/samples/"
4. Notice the "color" attribute

**Expected behavior**
The links should be displayed in medium-gray, when hovered they should change to a lighter gray (dark theme) and inversed in the light theme.

**Screenshots**
![image](https://github.com/nunocoracao/blowfish/assets/927896/b1467034-49c8-4b28-9faa-7d5f7fac5492)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Version: 114.0.1823.*

**Hugo & Blowfish versions**
Which version of Hugo and Blowfish are you using?
Using the latest version, this also affects the official documentation page "blowfish.page".

**Additional context**

**Comment by nunocoracao** [comment]
Hello @fuse314 why do you say this is the expected behavior?

**Comment by fuse314** [comment]
I started looking into this, because I find the top links hard to identify as links.
Most web pages indicate clickable links with a hover effect (color change or change in text-decoration). - Just look around github.com.
The included Tailwind class names ``text-gray-500`` and ``hover:text-gray-900`` indicate to me, that the text color is supposed to be gray and light gray on hover, however these tailwind class definitions are not included in the ``main.bundle.min.xx.css`` file.
The CSS rule that therefore is applied is the default "a" rule ``a{color:inherit;text-decoration:inherit}`` in ``main.bundle.min...css``. This inherits the text color set in the body rule, applying the variable ``--color-neutral``, that is defined as ``255,255,255`` (white) for the dark theme.
*(sorry for the long explanation)* 😉

Long story short: I think the usability of the theme would be improved by a "hover" color change on the top navigation links.
Maybe you can keep the current white color, but add a darker color on hover effect (for the dark theme).

**Comment by JahsehJ** [comment]
I haven't looked into which update got this fixed, however navbar links now have hover effects.
If that was what @fuse314 meant, we can close the issue.
